test/util/util.go

```diff
@@ -32,6 +32,8 @@ var (
 	CleanupTimeout       = time.Second * 5
 )
 
+const EmptyCurls = "{}"
```

Comment:
nit: emptyCurls 

i.e make this const package internal

Reply:
Done